#! /usr/bin/env python

PACKAGE = 'audio_to_spectrogram'

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("maximum_amplitude", double_t, 0, "Maximum range of amplitude to plot", 10.0)
gen.add("window_size", double_t, 0,  "Window size of data input to plot", 10.0)

exit(gen.generate(PACKAGE, PACKAGE, "DataAmplitudePlot"))
